app/uk/gov/hmrc/tai/viewModels/income/estimatedPay/update/CheckYourAnswersViewModel.scala

```diff
@@ -0,0 +1,119 @@
+    )
+
+    val hasDeductionConfirmationLine = createCheckYourAnswerConfirmationLine(
+      messages("tai.estimatedPay.update.checkYourAnswers.hasDeduction"),
+      Some(messages(s"tai.label.${hasDeductions.toLowerCase}")),
+      controllers.income.estimatedPay.update.routes.IncomeUpdateCalculatorController.payslipDeductionsPage().url
+    )
+
+    val taxablePayConfirmationLine = createCheckYourAnswerConfirmationLine(
+      messages("tai.estimatedPay.update.checkYourAnswers.taxablePay", timePeriod(paymentFrequency)),
+      taxablePay,
+      controllers.income.estimatedPay.update.routes.IncomeUpdateCalculatorController.taxablePayslipAmountPage().url,
+      isMonetaryValue
+    )
+
+    val hasBonusOrOvertimeConfirmationLine = createCheckYourAnswerConfirmationLine(
+      messages("tai.estimatedPay.update.checkYourAnswers.hasBonusOrOvertime"),
+      Some(messages(s"tai.label.${hasBonusOrOvertime.toLowerCase}")),
+      controllers.income.estimatedPay.update.routes.IncomeUpdateCalculatorController.bonusPaymentsPage().url
+    )
+
+    val hasExtraBonusOrOvertimeAnswer =
+      if (hasExtraBonusOrOvertime.isDefined) {
+        Some(messages(s"tai.label.${hasExtraBonusOrOvertime.get.toLowerCase}"))
+      }else{
+        hasExtraBonusOrOvertime
+      }
+
+    val hasExtraBonusOrOvertimeConfirmationLine = createCheckYourAnswerConfirmationLine(
+      messages("tai.estimatedPay.update.checkYourAnswers.hasExtraBonusOrOvertime"),
+      hasExtraBonusOrOvertimeAnswer,
+      controllers.income.estimatedPay.update.routes.IncomeUpdateCalculatorController.bonusPaymentsPage().url
+    )
+
+    val totalBonusOrOvertimeMessage =
+      if(hasExtraBonusOrOvertime.getOrElse("") == "Yes"){
+      messages("tai.estimatedPay.update.checkYourAnswers.totalYearlyBonusOrOvertime")
+     }else{
+        Messages("tai.estimatedPay.update.checkYourAnswers.totalBonusOrOvertime", timePeriod(paymentFrequency))
+      }
+
+    val totalBonusOrOvertimeConfirmationLine = createCheckYourAnswerConfirmationLine(
+      totalBonusOrOvertimeMessage,
+      totalBonusOrOvertime,
+      controllers.income.estimatedPay.update.routes.IncomeUpdateCalculatorController.bonusOvertimeAmountPage().url,
+      isMonetaryValue
+    )
+
+    Seq(paymentFrequencyConfirmationLine, totalPayConfirmationLine, hasDeductionConfirmationLine, taxablePayConfirmationLine,
+      hasBonusOrOvertimeConfirmationLine, hasExtraBonusOrOvertimeConfirmationLine, totalBonusOrOvertimeConfirmationLine).flatten
+  }
+
+  private def createCheckYourAnswerConfirmationLine(message: String, answer: Option[String], changeUrl: String,
+                                                    isMonetaryValue: Boolean = false)(implicit messages: Messages): Option[CheckYourAnswersConfirmationLine] = {
+
+    (answer,isMonetaryValue) match {
+      case (Some(_),true) => Some(CheckYourAnswersConfirmationLine(message,
+        Money.pounds(BigDecimal(answer.get)).toString().trim.replace("&pound;", "\u00A3"),
```

Comment:
What was the motivation for changing the encoding of the £ sign? Is there a utility that we can use that does this? If not we might want to check if this has been done elsewhere and create one.

Reply:
This was borrowed from a similar view model. I have updated the code to use a view model helper class which inserts the £ sign. As far as encoding is concerned we should be ok as the following is used at the top of the webpage: <meta http-equiv="content-type" content="text/html; charset=UTF-8" />